src/sinks/gcp/cloud_storage.rs

```diff
@@ -131,6 +131,10 @@ impl SinkConfig for GcsSinkConfig {
     fn sink_type(&self) -> &'static str {
         NAME
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
Should this be `true`? Again, I'm imagining that as long as the HTTP request is accepted, that it can be considered ack'd.

Reply:
I also think this should be true, from a quick look at the gcp docs.

Reply:
Fixed in 62b55dfeb